index.js

```diff
@@ -1,254 +1,380 @@
-const Discord = require('discord.js')
-const client = new Discord.Client()
+/* eslint-disable-file */
+const Discord = require("discord.js");
+const { ReactionRole } = require("reaction-role");
+const dotenv = require("dotenv");
 
-/* requirements to read bot token as environmental variable */
-const dotenv = require('dotenv'),
-      path   = require('path')
-dotenv.config({path: path.join(__dirname, '../.env')})
+dotenv.config();
+botSecretToken = process.env.BOT_TOKEN;
+botTwoSecretToken = process.env.BOT_TOKEN_TWO;
```

Comment:
Can you create a README.md file or some documentation for how are other developers to know how to set up the secrets, etc.

Reply:
ive added the readme, do give it a once over thanks